corehq/apps/auditcare/middleware.py

```diff
@@ -92,4 +92,7 @@ def _save_audit(self, request, response):
                     audit_doc.user = request.couch_user.username
             if response is not None:
                 audit_doc.status_code = response.status_code
-            audit_doc.save()
+            try:
+                audit_doc.save()
+            except Exception:
+                log.exception("error saving view audit")
```

Comment:
I'm glad you asked, because this question was in the back of my mind as well. My reading of how [`notify_exception`](https://github.com/dimagi/commcare-hq/blob/ec8433831ffbba0e8fa221c75f579d21dbac324d/corehq/ex-submodules/dimagi/utils/logging.py#L14) works led me to believe that all logging at ERROR level or above goes to Sentry, but maybe I'm wrong. @snopoke do you know?

Reply:
I used an interactive shell to verify that exceptions logged this way are sent to sentry. https://sentry.io/organizations/dimagi/issues/2321948885/